src/logging.rs

```diff
@@ -149,7 +171,7 @@ where
         .context("failed to open datagram socket")?;
 
     let mut data = Vec::new();
-    add_field_and_payload(&mut data, "PRIORITY", &(u8::from(priority)).to_string());
+    add_field_and_payload(&mut data, "PRIORITY", priority.as_str());
```

Comment:
I'd again like to ask whether this really matters; we're talking about a single byte after all…

That said, if a direct conversion to `str` is the way to go, do we still need the `From<u8>`  impl?